aucoalesce/coalesce.go

```diff
@@ -476,7 +476,7 @@ func applyNormalization(event *Event) {
 		default:
 			for _, n := range norms {
 				for _, f := range n.HasFields.Values {
-					if _, found := event.Data[f]; !found {
+					if _, found := event.Data[f]; !found { //nolint:staticcheck
```

Comment:
This is correctly found as incorrect.

Reply:
Thanks! I did not spot this or understand what it was tell me.

Reply:
staticcheck does a very deep analysis that sometimes surprises people with its insight. This is one of the very good reasons to include the reason in the linter directive; it stops you from blithely asking for an ignore and requires that you have an argument for why it's not right for the case.